bitnami/mongodb/values.yaml

```diff
@@ -1104,3 +1148,311 @@ metrics:
     ##       summary: High request latency
     ##
     rules: {}
+
+##
+## MongoDB(R) Hidden node parameters.
+##
+hidden:
+  ## Enable deploying the MongoDB(R) Hidden Node
+  ##   https://docs.mongodb.com/manual/tutorial/configure-a-hidden-replica-set-member/
+  ##
+  enabled: false
+
+  ## MongoDB(R) configuration file for Primary and Secondary nodes. For documentation of all options, see:
+  ##   http://docs.mongodb.org/manual/reference/configuration-options/
+  ##
+  configuration: ""
+
+  ## ConfigMap with MongoDB(R) configuration for Primary and Secondary nodes
+  ## NOTE: When it's set the arbiter.configuration parameter is ignored
+  ##
+  # existingConfigmap:
+
+  ## Command and args for running the container (set to default if not set). Use array form
+  ##
+  # command:
+  # args:
+
+  ## Additional command line flags
+  ## Example:
+  ## extraFlags:
+  ##  - "--wiredTigerCacheSizeGB=2"
+  ##
+  extraFlags: []
+
+  ## Additional environment variables to set
+  ## E.g:
+  ## extraEnvVars:
+  ##   - name: FOO
+  ##     value: BAR
+  ##
+  extraEnvVars: []
+
+  ## ConfigMap with extra environment variables
+  ##
+  # extraEnvVarsCM:
+
+  ## Secret with extra environment variables
+  ##
+  # extraEnvVarsSecret:
+
+  ## Annotations to be added to the MongoDB(R) statefulset. Evaluated as a template.
+  ##
+  annotations: {}
+
+  ## Additional labels to be added to the MongoDB(R) statefulset. Evaluated as a template.
+  ##
+  labels: {}
+
+  ## Number of MongoDB(R) replicas to deploy.
+  ## Ignored when mongodb.architecture=standalone
+  ##
+  replicaCount: 2
+
+  ## StrategyType for MongoDB(R) statefulset
+  ## It can be set to RollingUpdate or Recreate by default.
+  ##
+  strategyType: RollingUpdate
```

Comment:
```suggestion
  ## Annotations to be added to the MongoDB(R) Hidden statefulset. Evaluated as a template.
  ##
  annotations: {}

  ## Additional labels to be added to the MongoDB(R) Hidden statefulset. Evaluated as a template.
  ##
  labels: {}

  ## Number of MongoDB(R) Hidden replicas to deploy.
  ##
  replicaCount: 2

  ## StrategyType for MongoDB(R) Hidden statefulset
  ## It can be set to RollingUpdate or Recreate by default.
  ##
  strategyType: RollingUpdate
```